### PR TITLE
refactor: use modules in client

### PR DESCRIPTION
**Pull request by anshumanv**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [x] This is a **code refactor**
- [ ] This is a **test update**
- [ ] This is a **docs update**
- [ ] This is a **metadata update**

### For Bugs and Features; did you add new tests?
Exisitng

### Motivation / Use-Case
- Use module syntax in client

### Breaking Changes
no

### CODE REVIEW

**Comment by anshumanv** [comment]
@alexander-akait some error, can't figure out what's wrong 😞

File: client-src/clients/SockJSClient.js [hunk 1]
```diff
@@ -36,4 +34,6 @@ module.exports = class SockJSClient extends BaseClient {
       f(e.data);
     };
   }
-};
+}
+
+export default SockJSClient;
```
Comment (alexander-akait) [inline client-src/clients/SockJSClient.js:39]: We need load this clients using `import()`

Comment (anshumanv) [reply client-src/clients/SockJSClient.js:39]: we can just do `require(..).default` ?

they are imported in es5 code

**Comment by alexander-akait** [comment]
Already fixed

**Pull request closed without merging**